### DISCUSSION TITLE
Media Corps discussion: Points of Contact

### DISCUSSION THREAD

**Discussion by bernard0omnisend** [WordPress/Marketing-Team#404] [General]
As part of the new Media Corps at dotorg push mentioned by Josepha in the [post here](https://make.wordpress.org/marketing/2024/03/20/making-a-wordpress-media-corps/), and moving on from [this Slack thread](https://wordpress.slack.com/archives/C0GKJ7TFA/p1711012956711869), a few details need to be worked out in order to align folks and figure out what the next best steps are.

One important area is the Points of Contact question for the PR push:
Logically, we need points of contact for media inquiries. Who are the go-tos for different topics? Do we have a rotating list? Are they team leads, or the most senior/knowledgeable for a given area?

This discussion thread can serve as that place where the discussion happens, and to figure out how to move on to the next steps.

**Comment by karks88** [comment]
Hi, I chimed in on the Slack thread, but will summarize here as well.

I write about WordPress and often cover items related to new features. One of the challenges I face is finding someone to answer questions or provide further insight. I'm often limited to what I can find on .org.

If the source is from .org, I will sometimes try and contact the post author. It may be via Slack or social media. Sometimes, I get a response (and am thankful for it). However, that isn't always the case. I've shelved some articles because I couldn't get the info I needed.

I think it would be helpful to have an official process for requesting an interview. This saves me from having to figure out who to contact. And I'm hoping less stressful for a contributor is contacted out of the blue.

**Reply by bernard0omnisend** [reply]
Hi @karks88 thanks for chiming in! I'm hoping others will join in on this discussion.

I guess as a sort of summary:
What you'd be looking for is a sort of index/easy-to-read list of persons responsible for specific topics, and hopefully a dependable way to get in touch with them? Would a sheet of points of contact be enough? what kind of "process for requesting an interview" are you thinking of?

**Reply by karks88** [reply]
Hi @bernard0omnisend, I'm glad to help!

I think an index resource would be helpful. For example, if you want to speak to someone about a new Gutenberg feature (say, the new Font API), you're provided with a person to contact. A simple sheet could work for that.

I don't know how formal you'd want to get with things. For me, it's enough to have a person to contact. 

As far as the project is concerned, there may need to be a way to funnel requests or otherwise keep track of them. And possibly some privacy concerns for those willing to be contacted.

It could be that you have to fill out a form to request comment. That could be routed to the correct person based on the predefined subject. Or general email addresses that forward to that person.

I'm not sure how involved team members want to be with the process. But it could be anything from that simple contact list to something more robust.

**Comment by raewrites** [comment]
Hello! Adding my two cents (thanks for the heads up, @karks88!).

Agree with everything Eric said above. You may want to start with a media corps contributor who fields inquiries while processes are still being figured out, and until project contributors feel comfortable being contacted out of the blue with requests.

It's also worth thinking about contacts who can handle inquiries for different languages.

In Australia, some universities have a searchable list of experts, allowing journalists to quickly find someone to provide a quote for a timely news story. Maybe something to aspire to. It would be certainly helpful once the media corps is extended to supporting journalists outside of WordPress.

**Comment by bernard0omnisend** [comment]
Hey @raewrites and @karks88  thanks for your 2 cents. Very valuable!

And I think Rae your idea makes a lot of sense for where we are now in this Media Corps project: one Contributor who can handle incoming requests. Maybe that person can also (slowly) build up a POC list -- perhaps by asking for volunteers for people in various topics who would be wiling to handle requests.

In terms of timing, this would allow us to start the project soon without having necessarily a finished "index," but instead build up that index & that process based on how the Media Corps actually goes and what the emerging needs are.

What do you all think?

**Reply by raewrites** [reply]
Yep, agree.

It would be helpful to have some kind of "newsroom" landing page where media partners could access a media kit, latest press releases, contact information for the contributor handling inquiries, etc. I might be too close as a potential media partner, but happy to help where I can.

**Reply by karks88** [reply]
I think this makes a lot of sense as well. Seems better to start with one person and adjust as the needs become clearer.

**Reply by darijaomnisend** [reply]
Hey guys! In case you need someone to help with incoming requests or establishing a newsroom of sorts - I have some expertise in that sphere as I work in PR. However, I am kinda new in the community so would need some advice on how to organise this.
